core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1171,18 +1172,22 @@ public void executeSystemTask(WorkflowSystemTask systemTask, String taskId, int
                     task.setStatus(CANCELED);
                 }
                 executionDAOFacade.updateTask(task);
-                queueDAO.remove(QueueUtils.getQueueName(task), task.getTaskId());
+                queueDAO.remove(queueName, task.getTaskId());
                 return;
             }
 
             if (task.getStatus().equals(SCHEDULED)) {
                 if (executionDAOFacade.exceedsInProgressLimit(task)) {
                     //to do add a metric to record this
                     LOGGER.warn("Concurrent Execution limited for {}:{}", taskId, task.getTaskDefName());
+                    // Postpone a message with duration of 60 seconds, so that it would be available for poll again.
+                    queueDAO.postpone(queueName, taskId, task.getWorkflowPriority(), 60);
```

Comment:
Can we make this time duration configurable using a property? Overriding to a customizable value can provide more controls over the invisibility duration.

Reply:
Done.